parquet-tools/src/test/java/org/apache/parquet/tools/command/TestColumnSizeCommand.java

```diff
@@ -0,0 +1,88 @@
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.parquet.tools.command;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+import org.apache.parquet.example.data.simple.SimpleGroup;
+import org.apache.parquet.hadoop.ParquetWriter;
+import org.apache.parquet.hadoop.example.ExampleParquetWriter;
+import org.apache.parquet.hadoop.example.GroupWriteSupport;
+import org.apache.parquet.schema.MessageType;
+import org.apache.parquet.schema.PrimitiveType;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.Map;
+import java.util.Random;
+
+import static org.apache.parquet.schema.PrimitiveType.PrimitiveTypeName.INT32;
+import static org.apache.parquet.schema.PrimitiveType.PrimitiveTypeName.INT64;
+import static org.apache.parquet.schema.Type.Repetition.REQUIRED;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+public class TestColumnSizeCommand {
+
+  private final int numRecord = 10000;
+  private ColumnSizeCommand command = new ColumnSizeCommand();
+  private Configuration conf = new Configuration();
+
+  @Test
+  public void testColumnSize() throws Exception {
+    String inputFile = createParquetFile("input");
+    Map<String, Long> columnSizeInBytes = command.getColumnSizeInBytes(new Path(inputFile));
+    assertEquals(columnSizeInBytes.size(), 2);
+    assertTrue(columnSizeInBytes.get("DocId") > columnSizeInBytes.get("Num"));
+    Map<String, Float> columnRatio = command.getColumnRatio(columnSizeInBytes);
+    assertTrue(columnRatio.get("DocId") > columnRatio.get("Num"));
+  }
+
+  private String createParquetFile(String prefix) throws IOException {
+    MessageType schema = new MessageType("schema",
+      new PrimitiveType(REQUIRED, INT64, "DocId"),
+      new PrimitiveType(REQUIRED, INT32, "Num"));
+
+    conf.set(GroupWriteSupport.PARQUET_EXAMPLE_SCHEMA, schema.toString());
+
+    String file = createTempFile(prefix);
+    ExampleParquetWriter.Builder builder = ExampleParquetWriter.builder(new Path(file)).withConf(conf);
+    Random rnd = new Random();
+    try (ParquetWriter writer = builder.build()) {
+      for (int i = 0; i < numRecord; i++) {
+        SimpleGroup g = new SimpleGroup(schema);
+        g.add("DocId", rnd.nextLong());
+        g.add("Num", rnd.nextInt());
+        writer.write(g);
+      }
+    }
+
+    return file;
+  }
+
+  private static String createTempFile(String prefix) {
```

Comment:
Instead of having this file, we can use a JUnit rule: https://github.com/apache/parquet-mr/blob/c3e1a849d02042b324e4f72e1e4bba6c2a8da5bc/parquet-cli/src/test/java/org/apache/parquet/cli/commands/FileTest.java#L42

Then we can do: `temporaryFolder.newFile("test.parquet");`

Would be nice to keep this consistent

Reply:
Sounds good!